irksome/galerkin_stepper.py

```diff
@@ -51,69 +104,47 @@ def getFormGalerkin(F, L_trial, L_test, Q, t, dt, u0, stages, bcs=None):
     :arg u0: a :class:`Function` referring to the state of
          the PDE system at time `t`
     :arg stages: a :class:`Function` representing the stages to be solved for.
-    :arg bcs: optionally, a :class:`DirichletBC` object (or iterable thereof)
```

Comment:
Seems like we should update the doctoring for `Q` to `Qdefault`, with some explanation of what that means